lib/govuk_connect/cli.rb

```diff
@@ -917,18 +918,18 @@ def check_for_additional_arguments(command, args)
 
   def types
     @types ||= {
-      "app-console" => proc do |target, environment, args, extra_args, _options|
-        check_for_target(target)
-        check_for_additional_arguments("app-console", args)
-        check_for_additional_arguments("app-console", extra_args)
-        govuk_app_command(target, environment, "console")
+      "app-console" => proc do |target, _environment, _args, _extra_args, _options|
+        target ||= "<app-name>"
+        info "#{bold('The command for opening a Rails console has changed. Run this instead:')}\n\n"
+        info "  k exec -it deploy/#{target} -- rails c\n\n"
```

Comment:
This will generally work, but won't for all targets. For example Whitehall and draft apps require the machine class to prepended:

e.g.
```
k exec -it deploy/whitehall_backend/whitehall -- rails c
```

Reply:
Ohh, good catch, thanks!